ivy_tests/test_ivy/test_frontends/test_torch/test_vision_functions.py

```diff
@@ -6,6 +6,22 @@
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 
 
+# interpolation
+@handle_frontend_test(
```

Comment:
Hi there, can you please try to write a test function for this? It will help you to write the function too!